frontend/src/components/VCS/VCSProviderBasicInfoPanel.vue

```diff
@@ -157,7 +157,7 @@
           <li>• Workspace membership (Read)</li>
           <li>• Projects (Read)</li>
           <li>• Webhooks (Read and Write)</li>
-          <li>• Repositories (Read)</li>
+          <li>• Repositories (Read and Write)</li>
```

Comment:
Webhooks Write permission will force to have the Repositories Write permission @d-bytebase